### PR TITLE
Nw-class-4/mya skylar

### PR DESCRIPTION
**Pull request by myaskylar**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Mya Skylar
- Your City: North-west
- Your Slack Name: Mya skylar

# Homework Details

- Module: course-work-wk2
- Week: Week 3

# Notes

- What did you find easy?
  HTML structure.
- What did you find hard?
   To create responsive nav-bar
- What do you still not understand?
   Need more practice on media queries.
- Any other notes?
  I've notice to create the hamburger menu bar on mobile screen size needs js knowledge.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -16,4 +15,307 @@ body {
...
+}
+
+#toggle {
+  display: none;
+}
+
+/*styling article*/
+
+.intro-karma {
+  background-image: url(/img/first-background.jpg);
```
Comment (KarenPudner) [inline css/style.css:69]: The path to the image is not correct, so it is not displaying. Your CSS file is in a subfolder, so you need to go up a level before going into the img folder

File: index.html [hunk 2]
```diff
@@ -41,6 +41,21 @@ <h1>Introducing Karma</h1>
         <p>Bring WiFi with you, everywhere you go</p>
         <button>Learn More</button>
       </article>
+      <h1>Everyone needs a little Karma.</h1>
+      <section class="device">
```
Comment (KarenPudner) [inline index.html]: You don't seem to refer to the classes "device", "boost" or "pay" in your css file.

**Review by KarenPudner — APPROVED** [review]
Good work! The main issue is that the main image is not displaying - I've downloaded it locally, so I can see that it will display correctly if you just tweak the path. You generally target classes rather than IDs in your css, which is good. You also make good use of semantic HTML.  The images seem to respond well to changing the window size. It would be useful if you could include some screenshots of your webpage in different window sizes, and also run Lighthouse against it and include a screenshot in your solution. Let me know if you don't know how to do that.

File: css/style.css (hunk 1 above)
Comment (myaskylar) [reply css/style.css]: Thanks for the hint. I've correct the files accessibility.

File: index.html (hunk 2 above)
Comment (myaskylar) [reply index.html]: I've changed all section class to card and use it in .card CSS to style.

**Pull request closed without merging**